### DISCUSSION TITLE
Questions about un-escaped underscores in DOI field

### DISCUSSION THREAD

**Discussion by rpgoldman** [retorquere/zotero-better-bibtex#2548] [Q&A]
I have been having trouble with exporting to bibtex entries with DOIs that have underscores.  Latex gives me errors on this. But I was just looking at the discussion of #108 and in there it is stated that DOI underscores should _not_ be escaped.

So I'm a bit confused by this.  I will try to recreate a case and post it here if that will help. I encountered this while working on something with a tight deadline, so just ended up taking the DOIs out of the Zotero entries to avoid this problem.  Need to go back and figure out how to recreate it.  But it seemed like maybe this wasn't a bug and it might be something I'm doing wrong, or my bst file is doing wrong.

**Comment by retorquere** [comment]
bst means bibtex; bibtex does not support `doi` fields. The Better BibTeX exporter is erroneously exporting the `doi` field, I'll fix that.

**Reply by retorquere** [reply]
Until that is in the new release (which I'll likely cut today), you can add `bibtex.doi` to the skip fields in the export preferences.
